kalite/distributed/management/commands/setup.py

```diff
@@ -272,6 +272,8 @@ def handle(self, *args, **options):
         # Migrate the database
         call_command("syncdb", interactive=False, verbosity=options.get("verbosity"))
         call_command("migrate", merge=True, verbosity=options.get("verbosity"))
+        # Create *.json and friends database
+        call_command("syncdb", interactive=False, verbosity=options.get("verbosity"), database="topic_tools")
```

Comment:
Scratch that -- discussed in HipChat, and this will at least avoid errors for devs when no assessment items have been downloaded yet.